projects/core/src/checkout/store/checkout-state.ts

```diff
@@ -37,6 +37,7 @@ export interface CheckoutStepsState {
 
 export interface PaymentTypesState {
   entities: { [code: string]: PaymentType };
+  selected: string;
```

Comment:
If we store the selected type here, we end up storing it in 2 places (because I guess it's on the cart as well). 
What is the intention?

Reply:
The thing is I don't want to subscribe to the cart. Cart data is loaded many times during checkout; but this `selected` is usually not changed often.